src/Features/CSharp/Portable/Completion/CompletionProviders/TupleNameCompletionProvider.cs

```diff
@@ -74,11 +78,21 @@ private void AddItems(ImmutableArray<INamedTypeSymbol> inferredTypes, int index,
                     return;
                 }
 
+                // Note: the filter text does not include the ':'.  We want to ensure that if
+                // the user types the name exactly (up to the colon) that it is selected as an
+                // exact match.
+
                 var field = type.TupleElements[index];
-                var item = CommonCompletionItem.Create(
-                    field.Name + ":", _cachedRules, Glyph.FieldPublic);
+                var item = CommonCompletionItem.Create(field.Name + ColonString, _cachedRules, Glyph.FieldPublic, filterText: field.Name);
```

Comment:
use SymbolCompletionItem.  Then you don't need many of these arguments.  Once you do you an also pass in "insertionText".  And once you do that, you can get rid of GetTextChangeAsync :)